### PR TITLE
Fixing V3 interpreter for regression tests

### PR DESCRIPTION
**Pull request by linxuanm**
This PR aims to make the V3 interpreter pass the regression tests we have so far

### CODE REVIEW

File: src/engine/Debug.v3 [hunk 1]
```diff
@@ -4,7 +4,7 @@
 component Debug {
 	def paranoid = false;
 	def verbose = false;
-	def interpreter = false;
+	def interpreter = true;
```
Comment (titzer) [inline src/engine/Debug.v3:7]: This is left over.

**Comment by linxuanm** [comment]
TODO: check implicit popping after a `ctlxfer` with `resume`.

**Comment by linxuanm** [comment]
This PR is done; however, the checks do not pass as spectests do not pass for x86_64 target yet. Should we disable them for the time being (I don't think this can be done in `regress.sh` easily).

**Comment by titzer** [comment]
> This PR is done; however, the checks do not pass as spectests do not pass for x86_64 target yet. Should we disable them for the time being (I don't think this can be done in `regress.sh` easily).

Yes, it's possible to add them to `test/failures.x86-64-linux`.

**Pull request merged**